### PR TITLE
Sotatek teams create pull request!

### PR DESCRIPTION
**Pull request by Sotatek-TanHoang**

### CODE REVIEW

File: packages/common-solana/src/project/load.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export function loadFromJsonOrYaml(file: string): unknown {
+  const {ext} = path.parse(file);
+
+  if (ext !== '.yaml' && ext !== '.yml' && ext !== '.json') {
+    throw new Error(`Extension ${ext} not supported`);
+  }
+
+  const rawContent = fs.readFileSync(file, 'utf-8');
+  return yaml.load(rawContent);
+}
```
Comment (stwiname) [inline packages/common-solana/src/project/load.ts:18]: This function exists in `@subql/common`

File: packages/node-solana/package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "subql-node-solana": "./bin/run"
+  },
+  "dependencies": {
+    "@apollo/client": "3.5.8",
+    "@nestjs/common": "^8.2.6",
+    "@nestjs/core": "^8.2.6",
+    "@nestjs/event-emitter": "^1.0.0",
+    "@nestjs/platform-express": "^8.2.6",
+    "@nestjs/schedule": "^1.0.2",
+    "@polkadot/api": "7.8.1",
```
Comment (stwiname) [inline packages/node-solana/package.json:27]: This can be removed

**Review by stwiname — COMMENTED** [review]
It's proving to be quite hard to get changes from `packages/node` to `packages/node-solana`. To improve this we want to just have the one package `packages/node` and remove `packages/node-solana` but have the changes from solana applied to the main node package.

An example of missing data is https://github.com/subquery/subql-solana/blob/main/packages/node/src/configure/SubqueryProject.ts but there is currently no way to keep track of these changes. 

`common-solana` and `types-solana` can stay as is.

File: packages/node-solana/package.json (hunk 2 above)
Comment (Sotatek-TanHoang) [reply packages/node-solana/package.json:27]: I create a new pull request with node applied changes from node-solana. please check it out! 
https://github.com/subquery/subql-solana/pull/2

**Comment by Sotatek-TanHoang** [comment]
@stwiname 
I create a new pull request with node applied changes from node-solana. please check it out! 
https://github.com/subquery/subql-solana/pull/2

**Pull request merged**